easybuild/easyconfigs/g/g2clib/g2clib-1.6.3-foss-2020b.eb

```
@@ -0,0 +1,24 @@
+name = 'g2clib'
+version = '1.6.3'
+
+homepage = 'https://www.nco.ncep.noaa.gov/pmb/codes/GRIB2/'
+description = """Library contains GRIB2 encoder/decoder ('C' version)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

```suggestion
toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
```

and change the filename